internal/controller/dws_servers_controller.go

```diff
@@ -195,7 +195,7 @@ func (r *DWSServersReconciler) updateCapacityUsed(ctx context.Context, servers *
 
 		allocationSet := nnfStorage.Status.AllocationSets[storageIndex]
 
-		if allocationSet.Status != nnfv1alpha1.ResourceReady {
+		if allocationSet.Ready == false {
 			ready = false
```

Comment:
Couldn't you just set `false = allocation.Ready`?

Reply:
Yep... will change